tests/transformers/test_transformers_model_export.py

```diff
@@ -2062,6 +2122,62 @@ def test_qa_pipeline_pyfunc_predict(small_qa_pipeline):
     assert values.to_dict(orient="records") == [{0: "Run"}]
 
 
+@pytest.mark.parametrize(
+    ("input_image", "result"),
+    [
+        (os.path.join(pathlib.Path(__file__).parent.parent, "datasets", "cat.png"), False),
+        (image_url, False),
+        ("base64", True),
+        ("random string", False),
+    ],
+)
+def test_vision_is_base64_image(input_image, result):
+    if input_image == "base64":
+        input_image = base64.b64encode(read_image("cat_image.jpg")).decode("utf-8")
+    assert _TransformersWrapper.is_base64_image(input_image) == result
+
+
+@pytest.mark.parametrize(
+    "inference_payload",
+    [
+        [os.path.join(pathlib.Path(__file__).parent.parent, "datasets", "cat.png")],
+        [image_url, image_url],
+        "base64",
+    ],
+)
+@pytest.mark.skipif(
+    Version(transformers.__version__) < Version("4.33"), reason="skip these versions"
+)
```

Comment:
You should only apply this on "base64" parameter instead of skipping the whole test